INSTALLING.md

```diff
@@ -71,13 +71,23 @@ docker-compose up -d
 PostgreSQL and Redis will now set itself up in the background, and will be available in
 a minute or so.
 You'll need to run the `docker-compose up` command again if you
-reboot your computer to start the database again.
-
+reboot your computer to start the database again. If you like, you can run the
+container in the background by adding `-d` to the mentioned command.
 If you're already running a copy of PostgreSQL, you can use this too.
-You'll need to create a user with all privileges for the
-databases `koala-development` and `koala-test`, and set this in the `.env` file.
 
-There is an example file in the root of this repository called
+The previous command starts a process which should keep running while running Koala
+(unless you are part of the front-end, but it does enable hot-reloading. Choose for
+yourself). In addition, the following dependency should also be kept running:
```

Comment:
```suggestion
The next command starts a process which enabled hot-reloading for the javascript and css.
If you're not working on the front-end, you can skip this step.
```

The text suggested that the previous command (docker-compose up) is optional, which is not the case.